tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -729,10 +729,19 @@ public function preFlush(PreFlushEventArgs $eventArgs)
 
     public function testCommitWithExceptionClearsInProgressFlag() : void
     {
-        $listener = new class()
+        $listenerCallCounter = 0;
+        $listener = new class($listenerCallCounter)
         {
+            private $listenerCallCounter;
+
+            public function __construct(int &$listenerCallCounter)
```

Comment:
Please use public property instead of a reference.

Reply:
done